components/browser/thumbnails/src/main/java/mozilla/components/browser/thumbnails/storage/ThumbnailStorage.kt

```diff
@@ -42,6 +42,24 @@ class ThumbnailStorage(
         context.resources.getDimensionPixelSize(R.dimen.mozac_browser_thumbnails_maximum_size)
     private val scope = CoroutineScope(jobDispatcher)
 
+    /**
+     * Clears all the stored thumbnails in the disk cache.
+     */
+    fun clearThumbnails() {
+        logger.debug("Cleared all thumbnails from disk")
```

Comment:
This should be a in coroutine scope too.

Reply:
Fixed.